wakepy/_darwin.py

```diff
@@ -0,0 +1,31 @@
+from subprocess import Popen, PIPE
+
+COMMAND = u'caffeinate'
+ARGS = u" -d"
+BREAK = '\003'
+
+_process = None
+
+
+def set_keepawake(keep_screen_awake=False):
+    """
+    Set the keep-awake. During keep-awake, the CPU is not allowed to go to sleep
+    automatically until the `unset_keepawake` is called.
+
+    Parameters
+    -----------
+    keep_screen_awake: bool
+        If True, keeps also the screen awake.
+    """
```

Comment:
Shouldn't the `_process` be a global variable so that the module level `_process` value would be changed?

Reply:
The darwin implementation was tested in a slightly different context wherein I had classes.  I changed it to match your design and didn't realize I had to declare it as global.  I'll get that added and the conflicts resolved in just a moment.